scripts/pull_github_pr.sh

```diff
@@ -72,24 +73,21 @@ nr_commits=$(git log --pretty=oneline HEAD..FETCH_HEAD | wc -l)
 closes="${NL}${NL}Closes ${PROJECT}#${PR_NUM}${NL}"
 
 if [[ $nr_commits == 1 ]]; then
-	commit=$(git log --pretty=oneline HEAD..FETCH_HEAD | awk '{print $1}')
-	message="$(git log -1 "$commit" --format="format:%s%n%n%b")"
-	if ! git cherry-pick $commit
+	if ! git merge --log "$PR_REF"
```

Comment:
This is not git merge --squash. It will create a merge commit.

Reply:
it's rebase and merge, and will not create another merge commit.  we don't need the `--squash`

Reply:
I don't see --rebase.

Reply:
Sorry, i meant `--ff  = fast-forward`  but we should add `--ff-only`, so it will abort if fast-forward is not possible

Reply:
added

Reply:
Ok, with --ff-only it will show as merged when successful. But usually it will fail. If next has any commits after the branch-point of the patch, --ff-only will refuse to commit. We can ask the developer to rebase, but the rebase will be on master, not next. Next will usually be ahead.

Reply:
yes, you are right. So it seems like the wrong approach.

So probably `git merge --squash` is the right path